aspnetcore/tutorials/min-web-api.md

```diff
@@ -3,7 +3,7 @@ title: "Tutorial: Create a minimal API with ASP.NET Core"
 author: wadepickett
 description: Learn how to build a minimal API with ASP.NET Core.
 ms.author: wpickett
-ms.date: 05/05/2023
+ms.date: 09/05/2023
```

Comment:
```suggestion
ms.date: 5/05/2023
```

Reply:
> 

My bad. Thought it was necessary to change the date

Reply:
09 is the month, that's why it gave a build error.

`05/11/2023` would have been fine, but the date can't be more than  a week in the future.

Reply:
> 09 is the month, that's why it gave a build error.
> 
> `05/11/2023` would have been fine, but the date can't be more than  a week in the future.

That's why. I was using a different calendar format